### PR TITLE
Update deps and stuff

### PR DESCRIPTION
**Pull request by StefMa**
I've just updated all of our dependencies to the current latest version.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ before_script:
   - adb shell input keyevent 82 &
 
 script:
-  - ./gradlew build jacocoTestReport assembleAndroidTest connectedCheck
+  - ./gradlew clean build jacocoTestReport assembleAndroidTest connectedCheck
```
Comment (passsy) [inline .travis.yml]: why clean? It will delete all build artifacts from the `before_script` `assemble` step

File: build.gradle [hunk 2]
```diff
@@ -24,14 +25,19 @@ ext {
...
+    // Since composite is not up to date we can't use the latest one
+    compositeAndroidVersion = supportLibraryVersion
 
     bintrayVersion = '0.5.0'
     junitVersion = '4.12'
-    mockitoVersion = '1.10.19'
-    assertjVersion = '2.5.0'
+    mockitoVersion = '2.10.0'
+    assertjVersion = '2.8.0'
+    atslVersion = '1.0.1'
```
Comment (passsy) [inline build.gradle]: What does `atsl` stand for?

File: build.gradle [hunk 3]
```diff
@@ -24,14 +25,19 @@ ext {
...
-    BUILD_TOOLS_VERSION = '25.0.2'
+    TARGET_SDK_VERSION = 26
+    COMPILE_SDK_VERSION = 26
+    BUILD_TOOLS_VERSION = '26.0.1'
 
-    supportLibraryVersion = '25.2.0'
+    supportLibraryVersion = '26.0.0'
+    // compositeAndroidVersion should be always the same version as the supportLib version
+    // Since composite is not up to date we can't use the latest one
+    compositeAndroidVersion = supportLibraryVersion
```
Comment (passsy) [inline build.gradle]: good 👍

File: build.gradle (hunk 2 above)
Comment (passsy) [reply build.gradle]: android test support library? I suggest `supportTestVersion`

**Comment by StefMa** [comment]
I found that the travis.yml also includes `- build-tools-XX.X.X`...
Should we update these as well?
Then I will add a comment to the build-tools build.gradle as well.. Or isn't that needed? 🤔

**Comment by passsy** [comment]
Yes the same comment should be added to the build tools version

File: build.gradle [hunk 4]
```diff
@@ -24,14 +25,20 @@ ext {
...
-    TARGET_SDK_VERSION = 25
-    COMPILE_SDK_VERSION = 25
-    BUILD_TOOLS_VERSION = '25.0.2'
+    // If you change the SDK version don't forgot to update .travis.yml
+    TARGET_SDK_VERSION = 26
+    COMPILE_SDK_VERSION = 26
+    BUILD_TOOLS_VERSION = '26.0.1'
 
-    supportLibraryVersion = '25.2.0'
+    supportLibraryVersion = '26.0.0'
```
Comment (passsy) [inline build.gradle]: why not `26.1.0`?

Comment (StefMa) [reply build.gradle]: Because of https://github.com/grandcentrix/ThirtyInch/pull/115#discussion_r141189740 🙃

Comment (passsy) [reply build.gradle]: Good point!

**Pull request merged**